packages/mwp-app-server/src/util/getPlugins.js

```diff
@@ -20,23 +20,41 @@ import { plugin as requestAuthPlugin } from 'mwp-auth-plugin';
 const CSRF_COOKIE_NAME = 'x-mwp-csrf';
 const CSRF_HEADER_COOKIE_NAME = `${CSRF_COOKIE_NAME}-header`;
 const CSRF_HEADER_NAME = CSRF_COOKIE_NAME;
+// dev cookie names
+const DEV_CSRF_COOKIE_NAME = `${CSRF_COOKIE_NAME}-dev`;
+const DEV_CSRF_HEADER_COOKIE_NAME = `${DEV_CSRF_COOKIE_NAME}-header`;
+const DEV_CSRF_HEADER_NAME = DEV_CSRF_COOKIE_NAME;
 
+// Sets the csrf header token from plugin into a cookie
 export function setCsrfCookies(request, h) {
-	const csrfHeader = (request.response.headers || {})[CSRF_COOKIE_NAME];
+	const { headers = {} } = request.response;
+	const csrfHeader = headers[CSRF_COOKIE_NAME];
+	const csrfDevHeader = headers[DEV_CSRF_COOKIE_NAME];
```

Comment:
This function should only manage the cookie that is relevant to the environment, not both prod and dev - similar to the pattern we use for handling member cookies: https://github.com/meetup/meetup-web-platform/blob/master/packages/mwp-api-proxy-plugin/src/util/send.js#L26 - read/determine the cookie(s) of interest in one place and then continue execution without side effects on the other environment.

For these cookies, you can just choose the cookie based on `NODE_ENV === 'production'` rather than `api.isProd`

EDIT: see comment about assigning the `const`s initially to avoid any special logic here.